frontend/public/components/_masthead.scss

```diff
@@ -14,12 +14,30 @@
   @media(min-width: $grid-float-breakpoint) {
     height: $masthead-height-desktop;
   }
+  &__co-actions-menu {
+    padding: 0px 0px 0px 5px;
+  }
   &__console-picker {
     margin-left: 10px;
     @media(max-width: $grid-float-breakpoint-max) {
       display: none;
     }
   }
+  &__dropdowns {
+    display: flex;
+    justify-content: space-between;
+    margin-left: auto;
+    margin-right: 20px;
+  }
+  &__help {
+    @media (min-width: $grid-float-breakpoint) {
+      margin-right: 10px;
+    }
+  }
+  &__help-icon {
+    font-size: 14px;
```

Comment:
The icon doesn't render great at this font size (looks a bit anemic).  
![screen shot 2018-10-31 at 3 25 04 pm](https://user-images.githubusercontent.com/895728/47813347-37862e80-dd21-11e8-9905-c70764703a2e.PNG)